student.rb

```diff
@@ -4,7 +4,7 @@ class Student < Person
   attr_reader :classroom
 
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
-    super(age, name, parent_permission)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] Here it should be better if under age it actually displays the age of the student
![Screenshot from 2022-04-21 20-16-36](https://user-images.githubusercontent.com/62705947/164527565-acafa0a1-4403-400b-978b-c394f0eada29.png)